### ISSUE TITLE
Incorrect Return Value in readme

### ISSUE THREAD

**Issue by nekufa** [tarantool/tarantool-php#105]
Hello, i found that readme is incorrect.

There are a lot of comments like bool return "False and raises Exception on error".
It's impossible to return value and raise the exception.

What will happen? False will be returned or exception thrown?

**Comment by bigbes** [comment]
C API algorithm:
1) Exception variable set
2) return from function with `RETURN_FALSE`
tell me - what will happen?

**Comment by rybakit** [comment]
@bigbes If I get it right, the question is why a method returns a boolean value. What is the point of returning `bool true/false` if it will always be `true` on success and an exception on failure?

**Comment by bigbes** [comment]
@rybakit Is that a problem? Seems like something, that won't question anyone.

**Comment by rybakit** [comment]
@bigbes Imho, the problem that api gives a wrong impression that a user can just check the result of a method call, like:

```php
if (!$t->insert(...)) {
   // this block will never be reached
}
```

**Comment by bigbes** [comment]
It's not something, that bothers you before?

**Comment by rybakit** [comment]
@bigbes See this comment: https://github.com/tarantool/tarantool-php/issues/44#issuecomment-113064903

**Comment by bigbes** [comment]
Can you provide PR that will fix README.md?

**Comment by rybakit** [comment]
Of course, I can update README.md if you are OK to tweak the implementation.

**Comment by bigbes** [comment]
> What is the point of returning bool true/false if it will always be true on success and an exception on failure?

Implementation is OK, as I've understand, README gives wrong impression. Not?

**Comment by rybakit** [comment]
Hm, so you propose to modify the documentation to not reflect the real behavior?

**Comment by bigbes** [comment]
> What is the point of returning bool true/false if it will always be true on success and an exception on failure?

You have told, that this is the real behaviour, not?

**Comment by rybakit** [comment]
I didn't get your question, sorry. My point (although I'm not alone, as I'm not the one who created this issue /cc @nekufa) that mixing return booleans and exceptions are bad and I've already described and gave an example of why I think the api has the flaw. So, to fix that *both* the implementation *and* documentation should be updated to always return [void](http://php.net/manual/en/language.pseudo-types.php#language.types.void) and throw an exception in a case of an error. It doesn't have sense to me to only update the documentation and keep the current behavior. There is nothing much I can add to this discussion, but I hope I was clear enough ;)

P.S. I would really appreciate if you can provide me with the links to the docs or libraries which implements this "bool/exceptions" design, I have never seen this before. Maybe this will help me to understand your implementation.
